ui-speedspacechart/src/components/layers/SpeedLimitTagsLayer.tsx

```diff
@@ -1,21 +1,39 @@
-import React, { useEffect, useRef } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 
 import type { Store, tooltipInfos } from '../../types/chartTypes';
 import Tooltip from '../common/Tooltip';
 import { LINEAR_LAYERS_HEIGHTS } from '../const';
-import { drawSpeedLimitTags, computeTooltip } from '../helpers/drawElements/speedLimitTags';
+import {
+  drawSpeedLimitTags,
+  computeTooltip,
+  loadSpeedLimitTagsImages,
+} from '../helpers/drawElements/speedLimitTags';
 
 type SpeedLimitTagsLayerProps = {
   width: number;
   marginTop: number;
   store: Store;
 };
+type ImagesType = {
```

Comment:
Nit: in general I try to avoid putting "Type" in a type definition because it should already be clear from the naming that it's a type. Suggestion: `LoadedImages`?

Reply:
Done thanks!